readthedocs/storage/s3_storage.py

```diff
@@ -50,6 +50,10 @@ def __init__(self, *args, **kwargs):
                 'Ensure S3_BUILD_COMMANDS_STORAGE_BUCKET is defined.',
             )
 
+        self.bucket_acl = 'private'
+        self.default_acl = 'private'
+        self.querystring_auth = True
```

Comment:
We should make sure this works in dev, too.

Reply:
We could move the `S3PrivateBucketMixin` I added in https://github.com/readthedocs/readthedocs-corporate/pull/1103/files#diff-ec70ada13c82176ac8d16446079e8f0966583f9d20ec9c1a80b3b95ae0d19fafR8-R17 to this repository and share this code with .com

Reply:
yes, this works in dev. I think we can share a lot of code in .com now yes